hknweb/exams/views.py

```diff
@@ -12,13 +14,40 @@ def index(request):
 	return render(request, 'exams/index.html', context)
 
 def exams_for_course(request, department, number):
-	department = Department.objects.filter(name__exact=department).get()
+
+	department = Department.objects.filter(abbreviated_name__exact=department).get()
```

Comment:
Change this to just use `get`? (Or am I not actually correct?)
```suggestion
	department = Department.objects.get(abbreviated_name=department)
```

Reply:
Suggestion tested and works

Modified to use "get"